insights/parsers/auditd_conf.py

```diff
@@ -1,11 +1,25 @@
 """
-AuditdConf - file ``/etc/audit/auditd.conf``
-============================================
+Audit Conf files parsers
+========================
 
 The auditd.conf file is a standard key = value file with hash comments.
 Active settings are provided using the `get_active_settings_value` method or
 by using the dictionary `contains` functionality.
 
+The audispd.conf file has the same format and usage with auditd.conf.
+
+Note:
```

Comment:
`.. note::` would be betrer

Reply:
Thanks @xiangce , updated in newest commit.